lib/liquid/combinators/tags/case.ex

```diff
@@ -23,6 +23,13 @@ defmodule Liquid.Combinators.Tags.Case do
   alias Liquid.Combinators.{Tag, General}
   alias Liquid.Combinators.Tags.Generic
 
+  @type t :: [
```

Comment:
Shouldn't a Case type be just a tagged tuple instead of a list of tagged tuples?

Reply:
Every tag is a keyword list (which is a list of tagged tuples)